pyembed/src/interpreter_config.rs

```diff
@@ -21,14 +22,18 @@ use {
 
 #[cfg(target_family = "unix")]
 use std::{ffi::NulError, os::unix::ffi::OsStrExt};
+#[cfg(target_family = "unix")]
+type WcharT = i32;
```

Comment:
No need to get fancy. Can we please keep this named as `wchar_t`? This will require an inline annotation to `#[allow(..)]` whatever it needs to be to silence the compiler warning about type names.